admin/class-meta-columns.php

```diff
@@ -51,6 +62,20 @@ public function setup_hooks() {
 		}
 
 		add_filter( 'request', [ $this, 'column_sort_orderby' ] );
+
+		// Hook into tablenav to get the indexables.
+		add_action( 'manage_posts_extra_tablenav', [ $this, 'count_objects' ] );
+	}
+
+	/**
+	 * Retrieves the indexables for later use.
+	 *
+	 * @param string $target Extra table navigation location which is triggered.
+	 */
+	public function count_objects( $target ) {
```

Comment:
I find the name `count_objects` rather misleading. Maybe something with `cache` or `fetch`? The indexable part is also confusing, since it is the context. As with my comment on the `$indexables` variable.